### ISSUE TITLE
[Accessibility][CreateDashboardSuccessNotification] Dashboard Created Confirmation Message

### ISSUE THREAD

**Issue by mnischay** [awslabs/iot-application#1840]
####  Affected Groups: Screen Reader Users
 
Severity: High

 ####  Notes and Advice 
Visible status messages are apparent to sighted users, but not to people using a screen reader.
 
On submitting the "Create" dashboard form, the visually appearing status message "Successfully created dashboard "Test A11y"." is not announced when it appears on the screen.
 Recommendation 
Use role="status" for status notifications. When content is dynamically added to an element with this role, it will be announced by screen readers. Ensure that the status message is only announced once.
 
Status messages conveyed in this way must not set focus on the notification element.
 
Example code for defining a status region
 
<div role="status">
  <h2>Message sent</h2>
  <p>A copy has been saved in your Drafts folder.</p>
</div>
 
#### Note:
 Carefully consider when to use live regions, as announcements for frequently changing views can be disruptive and annoying. Take care not to overuse such announcements, and consider them the exception rather than the rule.
 
Assistive technologies will announce dynamic changes in the content of a live region, but the live region must first be present (and usually empty), so that the browser and assistive technologies are aware of it. Any subsequent changes are then announced. If the element with role="status" is only added at the point of notification, it will not be announced by assistive technologies. They respond to a change in the content of a live region, not a change in the existence of the live region itself.
 
#### Resources 
[ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)
[Defining 'Toast' Messages](https://adrianroselli.com/2020/01/defining-toast-messages.html)

### AC

- [ ] The above violation is mitigated
